core/src/main/scala/com/lightbend/paradox/markdown/StyledVerbatim.scala

```diff
@@ -65,5 +65,8 @@ abstract class StyledVerbatimSerializer extends VerbatimSerializer {
  */
 object PrettifyVerbatimSerializer extends StyledVerbatimSerializer {
   override def printPreAttributes(printer: Printer): Unit = printClass(printer, "prettyprint")
-  override def printCodeAttributes(printer: Printer, nodeType: String): Unit = printClass(printer, s"language-$nodeType")
+  override def printCodeAttributes(printer: Printer, nodeType: String): Unit = nodeType match {
+    case "nocode" => printClass(printer, s"nocode")
```

Comment:
Prettify will still try to highlight it. `nocode` is the only way to turn off this behaviour.